### ISSUE TITLE
Watch intervals should be run outside of main Angular zone

### ISSUE THREAD

**Issue by b-hobbs** [HackedByChinese/ng2-idle#40]
**I'm submitting a ...**  (check one with "x")
```
[ x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/HackedByChinese/ng2-idle/blob/master/CONTRIBUTING.md#getting-help
```

**Current behavior**

ng2-idle's watching uses setInterval on the main angular zone which is blocking Protractor because it waits until all calls in the main zone are finished.
 
**Expected behavior**

setInterval is run outside of the main angular zone so that it doesn't block Protractor. 

**Minimal reproduction of the problem with instructions**

You should be able to add protractor to an app setup for ng2-idle and use `protractor --elementExplorer` and try to grab an element through Protractor's api.

**Workaround**
Should preferably deal with this inside of the plugin itself.
```javascript
    this._ngZone.runOutsideAngular(() => {
      this.idle.watch();
    });

    this.idle.onTimeoutWarning.subscribe((countdown: number) => {
      this._ngZone.run(() => { this.sessionEndCountdown = countdown });
    });
```
**What is the motivation / use case for changing the behavior?**

Protractor tests don't work

**Please tell us about your environment:**

OSX EL Capitan 10.11.6
VS Code

* **@ng-idle version:** 2.x

2.0.0-beta.4
* **Angular version:** 2.x

2.4.7
* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

ALL
* **Language:** [all | TypeScript X.X | ES6/7 | ES5] 
ALL

* **Node:** `node --version` = 
7

**Comment by xtazy** [comment]
Same problem.

**Comment by santonica** [comment]
I provided a solution in [SO ](http://stackoverflow.com/questions/41611836/stop-ng2-idle-for-protractor-tests/43459183#43459183) for this and then found this open defect. Please fix this within the ng2-idle. Thx @b-hobbs for opening this issue.

**Comment by jmurphy87** [comment]
still experiencing this issue, any solutions yet?

**Comment by tiboprea** [comment]
Any updates on this issue?

**Comment by HackedByChinese** [comment]
Closed via #74

**Comment by veneliniliev** [comment]
i have same problem with version 7.0.0-beta.1
